zkchannels-contract/zkchannel_smartpy_script.py

```diff
@@ -341,25 +370,16 @@ def test():
     scenario.h3("mutualClose")
     scenario += c5.mutualClose(custBal = custBal, merchBal = merchBal, merchSig = merchSig).run(sender = aliceCust)
  
-    # scenario.h2("Scenario 6: escrow -> addCustFunding -> reclaimCustFunding")
-    # scenario.h3("escrow")
-    # c6 = ZkChannel(cid, aliceCust.address, bobMerch.address, aliceCust.public_key, bobMerch.public_key, custFunding, merchFunding, selfDelay, g2, merchPk0, merchPk1, merchPk2, merchPk3, merchPk4, merchPk5, close_flag)
-    # scenario += c6
-    # scenario.h3("Customer Funding their side of the channel")
-    # scenario += c6.addFunding().run(sender = aliceCust, amount = custFunding)
-    # scenario.h3("Customer pulling out their side of the channel (before merchant funds their side)")
-    # scenario += c6.reclaimFunding().run(sender = aliceCust)
-
-    scenario.h2("Scenario 7: Failing tests for custClose")
+    scenario.h2("Scenario 6: Failing tests for custClose")
     scenario.h3("escrow")
-    c7 = ZkChannel(cid, aliceCust.address, bobMerch.address, aliceCust.public_key, bobMerch.public_key, custFunding, merchFunding, selfDelay, g2, merchPk0, merchPk1, merchPk2, merchPk3, merchPk4, merchPk5, close_flag)
-    scenario += c7
-    scenario += c7.addFunding().run(sender = aliceCust, amount = custFunding)
+    c6 = ZkChannel(cid, aliceCust.address, bobMerch.address, aliceCust.public_key, bobMerch.public_key, custFunding, merchFunding, selfDelay, g2, merchPk0, merchPk1, merchPk2, merchPk3, merchPk4, merchPk5, close_flag)
+    scenario += c6
+    scenario += c6.addFunding().run(sender = aliceCust, amount = custFunding)
```

Comment:
Here and below. Don't try to manually number the test scenarios. It just means more work when things change. Instead use more descriptive names.

Reply:
renamed to abbreviated descriptive names